### DISCUSSION TITLE
Getting started with Wolf

### DISCUSSION THREAD

**Discussion by ConsularParadi** [games-on-whales/wolf#119] [Q&A] [answered]
Hi everyone, this seems like a very cool and difficult project to implement. May I know if there are any resources to start adding my own apps in the config such as Lutris? Also I have tried to launch the pre-configured Steam app but all I get is a sway desktop with terminal and steam does not launch. It would be nice if we as a community can help developers build a robust documentation so anyone can get started with the project.

**Comment by ABeltramo** [comment, marked as answer]
Hello there! 👋 

> May I know if there are any resources to start adding my own apps in the config such as Lutris?

The docs are lacking in that regard, unfortunately. There's a bit of info into how the Lutris image works [here](https://games-on-whales.github.io/gow/lutris.html) and how to build it from scratch and extend it. We are planning on adding more info to the docs but I had no time to actually do it myself.

> Also I have tried to launch the pre-configured Steam app but all I get is a sway desktop with terminal and steam does not launch

On first start it'll download the very first Steam client update and sometimes that view doesn't display correctly. Try to wait for a bit, if that still doesn't work feel free to open up an issue or jump on the Discord server and we'll be happy to help you out.

>  It would be nice if we as a community can help developers build a robust documentation so anyone can get started with the project

Absolutely, PRs are very welcome!!

**Reply by ConsularParadi** [reply]
Thank you for this amazing software you've built for us.
I'll start tinkering with the Lutris config and contribute to the docs as much as possible.
